list.json

```diff
@@ -1,146 +1,286 @@
 {
 	"corpora": {
 		"quora-duplicate-questions": {
-			"description": "over 400,000 lines of potential question duplicate pairs. Each line contains IDs for each question in the pair, the full text for each question, and a binary value that indicates whether the line truly contains a duplicate pair.",
+			"num_rows": 404290,
+			"row_format": "dict",
+			"size": "21684784 (~21MB)",
+			"reader": "https://github.com/RaRe-Technologies/gensim-data/releases/download/quora-duplicate-questions/__init__.py",
+			"license": "probably https://www.quora.com/about/tos",
+			"fields": {
```

Comment:
Too much detail; detailed format description belongs to the code example (`reader`, which could probably be better called `reader_code` or `format_example`).

Reply:
The description of the fields is very convenient and immediately gives a lot of information about the dataset (you do not need to load it), as for me this is really useful feature (the most useful information)

Reply:
rename reader 9c93d55

Reply:
Yeah, you shouldn't have to download a dataset to understand the format. But link to a concrete description (no download necessary) in the reader, with code too, sounds preferable -- why duplicate the work?

It's better to have all this information in one place, and just link to it.

Reply:
Fields invisible in the reader code (typically something like `yield json.loads(line)`, it makes no sense to list them, since the data sets are already prepared for use) + reader files has no "view" mode, i.e. if you insert link to your browser - you'll download this reader.

This one place is `lists.json`, available on GitHub as file or through `api.info(...)` call.